### PR TITLE
Fix ONNX export for AveragePool with count_include_pad=True

### PR DESCRIPTION
**Pull request by bddppq**
Currently ONNX AveragePool doesn't include padded pixels when calculating the edge values.

### CODE REVIEW

File: torch/onnx/symbolic.py
Comment (houseroad) [inline torch/onnx/symbolic.py]: Also add avg_pool1d, low hanging fruit.

Comment (bddppq) [reply torch/onnx/symbolic.py]: Done

**Review by houseroad — APPROVED** [review]

**Comment by bddppq** [comment]
@ezyang @soumith github requires review from you folks before allowing me to merge this one

**Review by soumith — APPROVED** [review]

**Pull request merged**
